markdown/ranger/ranger-kerberos.html.md.erb

```diff
@@ -0,0 +1,209 @@
+- Updating the HAWQ Ranger Plug-in Service configuration to utilize Kerberos
+- Updating the HAWQ service definition in the Ranger Admin UI
+
+
+### Procedure <a id="kerb_rps2ra_proc"></a>
+
+Perform the following steps to enable Ranger Plug-in Service policy refresh from a kerberized Ranger:
+
+1. Create a `postgres` Ranger administrative user.
+
+    1. Open the Ranger Admin UI. The default URL is \<ranger-admin-node\>:6080.
+    2. Navigate to **Settings > Users/Groups**.
+    3. Click the **Add New User** button.
+    4. Fill out the required fields in the **User Detail** dialog:
+
+        **User Name*** : Enter `postgres`.  
+        **New Password*** : Enter the user's password.  
+        **Password Confirm*** : Enter the password again.  
+        **First Name*** : Enter the first name of the user. 
+        **Select Role*** : Select the **Admin** role.
+
+2. Update the HAWQ Ranger Plug-in Service configuration to use Kerberos authentication when communicating with Ranger.
+
+    2. Edit the HAWQ Ranger Plug-in Service `rps.properties` configuration file and set the `RPS_AUTH` property to specify Kerberos authentication.
+
+        ``` shell
+        gpadmin@master$ vi $GPHOME/ranger/etc/rps.properties
+        ```
+    
+        ``` pre
+        RPS_AUTH=kerberos
+        ``` 
+
+    3. `rps.properties` also includes `RPS_PRINCIPAL` and `RPS_KEYTAB` properties. If these properties are unset, the Ranger Plug-in Service uses the cached Kerberos credentials that are available for the HAWQ runtime user account (typically `gpadmin`). Set these properties if the default values are not appropriate for your environment.
+
+3. Stop the Ranger Plug-in Service:
+
+    ``` shell
+    gpadmin@master$  $GPHOME/ranger/bin/rps.sh stop
+    ```
+
+4. Generate a ticket for the `postgres@<realm>` principal:
+
+    ``` shell
+    gpadmin@master$ kinit -kt /etc/security/keytabs/hawq.service.keytab postgres@REALM.DOMAIN
+    ```
+
+5. Restart the HAWQ Ranger Plug-in Service:
+
+    ``` shell
+    gpadmin@master$  $GPHOME/ranger/bin/rps.sh start
+    ```
+
+6. If you have enabled HAWQ Ranger High Availability, synchronize the HAWQ Ranger Plug-in Service configuration files from the HAWQ master node to the standby master node as described in the [Register a Standby Ranger Plug-in Service](ranger-integration-config.html#rps_ha_cfg_standbyrps) procedure, including restarting the standby master node.
+
+7. Navigate to the Ranger Admin UI in a supported web browser. The default URL is \<ranger-admin-node\>:6080. 
+
+8. Locate the **HAWQ** service definition and click the **Edit** button. 
+
+9. Update the applicable **Config Properties** fields:
+  
+    **HAWQ Authentication Type** : Select **Kerberos** from the drop-down list.  
+    **HAWQ Kerberos Service Name** : Enter `postgres` or your custom HAWQ service name.   
+
+10. **Save** your changes to permanently configure Kerberos authentication for Ranger policy retrieval.
```

Comment:
thanks for reviewing, @interma!

i had thought that:
- username and password were used when configuring ranger->kerberized hawq
- authentication type and service name were used when configuring RPS->kerberized ranger

if all four of them are only used for ranger->kerberized hawq, i will update the appropriate sections.